packages/kbn-es/src/integration_tests/__fixtures__/es_bin.js

```diff
@@ -87,11 +87,11 @@ const { ES_KEY_PATH, ES_CERT_PATH } = require('@kbn/dev-utils');
     }
   );
 
-  // setup server auto close after 1 second of silence
+  // setup server auto close after 5 second of silence
   let serverCloseTimer;
   const delayServerClose = () => {
     clearTimeout(serverCloseTimer);
-    serverCloseTimer = setTimeout(() => server.close(), 1000);
+    serverCloseTimer = setTimeout(() => server.close(), 5000);
```

Comment:
I really wish we could take the opportunity to refactor this code when it turns out to be flaky so that it doesn't rely on a timer ☹️ But maybe that's not easy in this case